### PR TITLE
Draft release notes for 12/10 ahead of 12/09 press release.

### PR DESCRIPTION
**Pull request by james-vendia**

### CODE REVIEW

File: releases/2021-12-10-releasenotes.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
 ---
-title: '<Placeholder Title>'
-description: 'TBD'
+title: 'A Sleigh Full of New Features'
+description: 'mart contracts, a pay-as-you-go pricing plan, the beta release of Azure support, and more!'
```
Comment (brian-vendia) [inline releases/2021-12-10-releasenotes.md:3]:
```suggestion
description: 'Smart contracts, a pay-as-you-go pricing plan, the beta release of Azure support, and more!'
```

File: releases/2021-12-10-releasenotes.md [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
-## Starter Tier
-### Starter Tier Limit Enforcement
-## Individual Tier
+The Vendia team has been hard at work building out new features that we know will bring cheer to our users.  We've been so busy and have so much to share that we've also created a [blog post](https://www.vendia.net/blog/announcing-new-features) to delve into more details about each new feature.
+
 ## Smart Contracts
-## Azure Support
\ No newline at end of file
+
+Vendia Share makes it easy to share data and files among participants in a Uni.  But sometimes data alone isn't enough.  Smart Contracts allow users to act on data in a prescribed way, as data changes across partners, clouds, and regions, without having to build or maintain a complex eventing infrastructure.
```
Comment (brian-vendia) [inline releases/2021-12-10-releasenotes.md:18]:
```suggestion
Vendia Share makes it easy to share data and files among participants in a Uni.  But sometimes _sharing_ data alone isn't enough.  Smart Contracts allow users to _take action_ on data in a prescribed way, as data changes across partners, clouds, and regions, without having to build or maintain a complex eventing infrastructure.
```

**Review by brian-vendia — CHANGES_REQUESTED** [review]
Minor changes suggested - looks really good!

**Review by brian-vendia — APPROVED** [review]
Looks good to me.

Question for @DavidWells - will the release nodes go live if we merge today (2021-12-09) and a post has a date of tomorrow (2021-12-10)? Assuming yes but please confirm. If so, I think it makes sense to merge tomorrow.

File: releases/2021-12-10-releasenotes.md [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
-## Azure Support
\ No newline at end of file
+
+Vendia Share makes it easy to share data and files among participants in a Uni.  But sometimes _sharing_ data alone isn't enough.  Smart Contracts allow users to _take action_ on data in a prescribed way, as data changes across partners, clouds, and regions, without having to build or maintain a complex eventing infrastructure.
+
+![Smart Contracts](https://user-images.githubusercontent.com/92179243/145060082-56557805-7708-4a26-8818-8dcfb95ceb78.png)
+
+Please see the new [Smart Contracts docs](https://www.vendia.net/docs/share/smart-contracts) for more information on how to use this powerful new feature and for more information on how Vendia Smart Contracts compare with smart contracts in other solutions.
+
+## Transactions
```
Comment (db-vendia) [inline releases/2021-12-10-releasenotes.md:24]: In the PR, we are calling this User-Level Transactions

File: releases/2021-12-10-releasenotes.md [hunk 4]
```diff
@@ -1,20 +1,38 @@
...
+
 ## Smart Contracts
-## Azure Support
\ No newline at end of file
+
+Vendia Share makes it easy to share data and files among participants in a Uni.  But sometimes _sharing_ data alone isn't enough.  Smart Contracts allow users to _take action_ on data in a prescribed way, as data changes across partners, clouds, and regions, without having to build or maintain a complex eventing infrastructure.
+
+![Smart Contracts](https://user-images.githubusercontent.com/92179243/145060082-56557805-7708-4a26-8818-8dcfb95ceb78.png)
+
+Please see the new [Smart Contracts docs](https://www.vendia.net/docs/share/smart-contracts) for more information on how to use this powerful new feature and for more information on how Vendia Smart Contracts compare with smart contracts in other solutions.
```
Comment (db-vendia) [inline releases/2021-12-10-releasenotes.md:22]: This link still 404's. When are these links going live?

**Review by db-vendia — COMMENTED** [review]
Minor changes nit picks. Some of the links don't work. Are those going live soon?

File: releases/2021-12-10-releasenotes.md [hunk 5]
```diff
@@ -1,20 +1,38 @@
...
+
+## Transactions
+
+The new Vendia Transactions feature enables users to group multiple changes on multiple data values in a single transaction and guarantees that all updates are processed serially, in order, as an atomic unit. For those looking for an analog to relational database transactions in Share, this feature is for you!  With this feature, Share takes a large step toward combining ACID semantics with a distributed serverless ledger.  For more information on how to apply Vendia Transactions, [see the docs](https://www.vendia.net/docs/share/vendia-transaction).
```
[the remark this answers is not in the document]
Comment (james-vendia) [reply releases/2021-12-10-releasenotes.md]: Correct.  Waiting for Brian's PR to get merged for this one and then this link will resolve properly.

File: releases/2021-12-10-releasenotes.md (hunk 4 above)
Comment (james-vendia) [reply releases/2021-12-10-releasenotes.md:22]: Correct.  Waiting for Rory's PR to get merged for this one and then this link will resolve properly.

**Pull request merged**